hx711_rpio.go

```diff
@@ -0,0 +1,128 @@
+// +build !windows,rpio
+
+package hx711
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/stianeikeland/go-rpio/v4"
+)
+
+// HostInit opens /dev/gpiomem. This needs to be done before Hx711 can be used.
+func HostInit() error {
+	return rpio.Open()
+}
+
+// NewHx711 creates new Hx711.
+// Make sure to set clockPin and dataPin to the correct pins.
+// https://cdn.sparkfun.com/datasheets/Sensors/ForceFlex/hx711_english.pdf
+func NewHx711(clockPinStr string, dataPinStr string) (*Hx711, error) {
+	clockPin, err := strconv.ParseInt(clockPinStr, 10, 32)
```

Comment:
Should this be 64 instead of 32?

Reply:
I don't think that 64 bit is a requirement for GPIO pin numbering.